src/contracts/core/AVSDirectory.sol

```diff
@@ -149,40 +425,132 @@ contract AVSDirectory is
      *
      */
 
+    /// @notice Returns operator sets an operator is registered to in the order they were registered.
+    /// @param operator The operator address to query.
+    /// @param index The index of the enumerated list of operator sets.
+    function operatorSetsMemberOf(address operator, uint256 index) public view returns (OperatorSet memory) {
```

Comment:
It feels like the CLI would want to just return the list of operator sets that the operator is in vs having to call to get the total and then calling with each index or paging through them

Reply:
There's a pagination fn that was just added for this `operatorSetsMemberOf(operator, start, length)`, pagination is needed to avoid gas limit related issues (if wanted).

Reply:
Even paging through them doesn't feel ideal.  I think it's okay to assume there won't be 1000s of operator sets for an operator